approval_polls/templates/base.html

```diff
@@ -47,7 +48,6 @@
               </p>
             </div>
           </div>
-
```

Comment:
The newline is there so it's easy to visually distinguish rows, there's no need to remove it.